Makefile

```diff
@@ -96,6 +96,11 @@ beats-dashboards:
 	mkdir -p build/dashboards
 	$(foreach var,$(BEATS),cp -r $(var)/_meta/kibana/ build/dashboards/$(var)  || exit 1;)
 
+# Collects all dashboards and generates dashboard folder for https://github.com/elastic/beats-dashboards/tree/master/dashboards
+.PHONY: import-dashboards
+import-dashboards: beats-dashboards
```

Comment:
I was going after what the doc said about `make import-dashboards`, maybe I should fix that instead then:

> If Elasticsearch is running on localhost, then you can run the following command from the Beat repository